src/TLuaInterpreter.cpp

```diff
@@ -3000,26 +3000,26 @@ int TLuaInterpreter::setBackgroundImage(lua_State* L)
     return 0;
 }
 
-int TLuaInterpreter::setLabelClickCallback(lua_State* L)
+int TLuaInterpreter::setLabelCallback(lua_State* L, QString funcName)
```

Comment:
I _think_ you can safely pass by `const` reference here - i.e. change the signature to:

    int TLuaInterpreter::setLabelCallback(lua_State* L, const QString& funcName)

which seems to be the _done_ thing - though, as it isn't modified the saving of a normal copy of  argument to method/function on entry is probably marginal given the inherently shared nature of `QString`s (they are a sort of "smart" class with _implicit sharing_ - once you have one instance it can be trivially and easily copied and you only pay the price for marking a copy if you modify one of the copies - a.k.a. _Copy-On-Write_).